base/Dockerfile.rhel

```diff
@@ -1,12 +1,16 @@
 FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-base-openshift-4.6
+
+# ubi8 does not have a python binary in PATH. Installing it explicitly.
+
 RUN INSTALL_PKGS=" \
       which tar wget hostname shadow-utils \
       socat findutils lsof bind-utils gzip \
-      procps-ng rsync iproute \
+      procps-ng rsync iproute python3 \
       " && \
     if [ ! -e /usr/bin/yum ]; then ln -s /usr/bin/microdnf /usr/bin/yum; fi && \
     echo 'skip_missing_names_on_install=0' >> /etc/yum.conf && \
     yum install -y --setopt=tsflags=nodocs ${INSTALL_PKGS} && \
+    alternatives --set python /usr/bin/python3 && \
```

Comment:
If this needs to be buildable for both 7 and 8 (ugh, why?), then something like:
        ( test -x /usr/bin/python || alternatives --set python /usr/bin/python3) && \

Reply:
ugh, but that would mean that we're installing python3 alongside python2 in base-7